PULL_REQUEST_TEMPLATE

```diff
@@ -1,24 +1,32 @@
 Jira: [JIRA-TOKEN](https://ticketevolution.atlassian.net/browse/JIRA-TOKEN)
 
+> [!IMPORTANT]
+> Toggle to SQUASH AND MERGE when merging PRs to `staging` branch
+
 ## Why / What
+Why: These changes are for...
+
+What: This PR adds/removes/fixes... 
 
-This PR adds/removes/fixes&hellip;
+If this PR requires other developers to make changes to their local environment after this PR has been merged, please
+describe here. Examples: DB migration, Docker container restart/rebuild, .env changes etc.
```

Comment:
Nobody is going to read this, and it will be missed by every team member who isn't doing the review. Team communication pertaining would likely be better handled on slack

Reply:
Still, but I think it's useful to have here for anyone looking at recent merged PRs too see what had changed

Reply:
Removed this since the checklist communicates the same thing